pkgs/development/tools/jpexs/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "jpexs";
-  version = "18.4.1";
+  version = "20.1.0";
 
   src = fetchzip {
     url = "https://github.com/jindrapetrik/jpexs-decompiler/releases/download/version${version}/ffdec_${version}.zip";
-    sha256 = "sha256-aaEL3xJZkFw78zo3IyauWJM9kOo0rJTUSKmWsv9xQZ8=";
+    sha256 = "sha256-fLG29PsLJkuCMu0kXHWdFg0Lnvw79O1nQKNVqw2PJBA=";
```

```suggestion
    hash = "sha256-ytGtylhyNSdKfuPclZRJasOb/cskW65hMd4NM/q+/Ko=";
```
Try changing the hash here from the `specified` one in the error to the `got` one.

The `sha256` -> `hash` change is unrelated (we're trying to replace `sha256` with `hash` in nixpkgs)